profiles/ldap.py

```
@@ -329,6 +332,11 @@ def ldap_get_roomnumber(account):
     except AttributeError:
         return ""
 
+def ldap_get_calendar(account):
+    try:
+        return account.icallink
+    except KeyError:
```

This smells like it fails silently. Maybe log this?